### ISSUE TITLE
Possible incorrect assert for unit test in SA1625.

### ISSUE THREAD

**Issue by dlemstra** [DotNetAnalyzers/StyleCopAnalyzers#2270]
The test `VerifyThatWhitespacesAreNormalizedForEmptyXmlElementsAsync` reports an issue here: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1625UnitTests.cs#L98. Additional white space should be ignored but `NormalizeWhitespace` on `XmlEmptyElementSyntax` adds extra spaces:
```
emptyXmlElement.ToString()
"<see cref=""TestClass""/>"
emptyXmlElement.NormalizeWhitespace().ToString()
"<see cref = ""TestClass""/>"
 ```
And that is why this unit tests marks it as incorrect. I wonder if we should report a diagnostic for that line. Can you confirm that this is unit tests is correct or should it be fixed? Once this has been fixed or marked as being correct I can continue working on #SA1642.

**Comment by vweijsters** [comment]
The original StyleCop accepts `<see cref = "TestClass"/>` without warnings, in fact it accepts any amount of spaces there. 

When editing in Visual Studio, the code snippet for `<see` inserts it without spaces, which is my preference as well. I would have no problem with also allowing a single space surrounding the '=' token, multiple spaces would be a no go for me.
